src/installer.js

```diff
@@ -331,7 +331,12 @@ var createPixmapIcon = function (options, dir, callback) {
  */
 var createHicolorIcon = function (options, dir, callback) {
   async.forEachOf(options.icon, function (icon, resolution, callback) {
-    var iconFile = path.join(dir, 'usr/share/icons/hicolor', resolution, 'apps', options.name + '.png')
+    var iconFile
```

Comment:
We should start using `let`.

Reply:
Agreed, but I think that should be different PR since `var` is used everywhere right now